src/aind_metadata_mapper/open_ephys/camstim_ephys_session.py

```diff
@@ -113,7 +124,17 @@ def __init__(self, session_id: str, json_settings: dict) -> None:
 
         self.available_probes = self.get_available_probes()
 
-    def generate_session_json(self) -> Session:
+    def run_job(self):
```

Comment:
The run_job should return a JobResponse type. You can probably call the GenericEtl _load method instead of overwriting it.